doc/source/train/examples/pytorch/distributing-pytorch/README.md

```diff
@@ -0,0 +1,442 @@
+            download=True,
+            transform=transform,
+        )
+
+    # Create data loaders
+    train_dataloader = DataLoader(training_data, batch_size=batch_size, shuffle=True)
+    test_dataloader = DataLoader(testing_data, batch_size=batch_size)
+
+    return train_dataloader, test_dataloader
+```
+
+Now, define a function that runs the core training loop. Note how the code synchronously alternates between training the model for one epoch and then evaluating the its performance.
+
+
+```python
+def train_func(config: Dict):
+    lr = config["lr"]
+    epochs = config["epochs"]
+    batch_size = config["batch_size_per_worker"]
+
+    # Get dataloaders inside the worker training function
+    train_dataloader, test_dataloader = get_dataloaders(batch_size=batch_size)
+
+    model = VisionTransformer(
+        image_size=32,   # CIFAR-10 image size is 32x32
+        patch_size=4,    # Patch size is 4x4
+        num_layers=12,   # Number of transformer layers
+        num_heads=8,     # Number of attention heads
+        hidden_dim=384,  # Hidden size (can be adjusted)
+        mlp_dim=768,     # MLP dimension (can be adjusted)
+        num_classes=10   # CIFAR-10 has 10 classes
+    )
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+    model.to(device)
+
+    loss_fn = nn.CrossEntropyLoss()
+    optimizer = torch.optim.AdamW(model.parameters(), lr=lr, weight_decay=1e-2)
+
+    # Model training loop
+    for epoch in range(epochs):
+        model.train()
+        for X, y in tqdm(train_dataloader, desc=f"Train Epoch {epoch}"):
+            X, y = X.to(device), y.to(device)
+            pred = model(X)
+            loss = loss_fn(pred, y)
+
+            optimizer.zero_grad()
+            loss.backward()
+            optimizer.step()
+
+        model.eval()
+        test_loss, num_correct, num_total = 0, 0, 0
+        with torch.no_grad():
+            for X, y in tqdm(test_dataloader, desc=f"Test Epoch {epoch}"):
+                X, y = X.to(device), y.to(device)
+                pred = model(X)
+                loss = loss_fn(pred, y)
+
+                test_loss += loss.item()
+                num_total += y.shape[0]
+                num_correct += (pred.argmax(1) == y).sum().item()
+
+        test_loss /= len(train_dataloader)
+        accuracy = num_correct / num_total
+
+        print({"epoch_num": epoch, "loss": test_loss, "accuracy": accuracy})
+```
+
+Finally, run training.
+
+
+```python
+def train_cifar_10():
+    global_batch_size = 512
+
+    train_config = {
+        "lr": 1e-3,
+        "epochs": 1,
+        "batch_size_per_worker": global_batch_size,
+    }
+
+    train_func(train_config)
+
+
+if __name__ == "__main__":
+    train_cifar_10()
+```
+
+If everything works as expected, the training should take around 2 minutes 10 seconds with an accuracy of around 0.35.  
+
+## Step 2: Distribute it to multiple machines with Ray Train and inspect results with the Ray Train Dashboard
```

Comment:
nit: I don't know if you need to explicitly call out Ray Train Dashboard in this header.